internal/http/certificate.go

```diff
@@ -7,6 +7,9 @@ import (
 )
 
 func LoadCertPool(path string) (*x509.CertPool, error) {
+	if path == "" {
+		return nil, nil
+	}
```

Comment:
Although it makes sense in `login.go`, it is weird to return a nil pool when err is nil. I suggest putting this logic back to the `login.go`.